addons/web/static/src/core/utils/misc.js

```diff
@@ -26,3 +26,21 @@ export function markEventHandled(ev, markName) {
     }
     eventHandledWeakMap.get(ev).push(markName);
 }
+
+/**
+ * Iterate through all parent elements and find the parent where selector
+ * matches and return all those parents.
+ * This method is similar to parents method of jQuery.
+ *
+ * @param {HTMLElement} el
+ * @param {string} selector
+ */
+export function parents(el, selector) {
```

Comment:
Similarly to https://github.com/odoo/odoo/pull/167137, we do not want to replicate the old JQuery methods.
Besides, the actual call in https://github.com/odoo/enterprise/pull/62467 does not even use the selector parameter - it just want the list of parents.

Reply:
I do agree, then no need of this PR.